.github/workflows/build.yml

```diff
@@ -0,0 +1,48 @@
+name: Build External Images
```

Comment:
Change to `name: Build CSI Sidecar Images` and rename the action file to `build-csi-sidecar-images`. 

Because we might have more external image builds in the future, we can make the build action workflow more specific.